src/components/ComposerAttachments.vue

```diff
@@ -157,8 +148,15 @@ export default {
 
 				return uploadLocalAttachment(file, progress(file.name)).then(({ file, id }) => {
 					logger.info('uploaded')
-					return this.emitNewAttachments([this.fileNameToAttachment(file.name, file.size, id)])
+					return this.emitNewAttachments([{
+						fileName: file.name,
+						displayName: trimStart('/')(name),
```

Comment:
`name` is undefined. this broke local attachments

Reply:
yes I'd discovered that too. but you fix it already I believe